packages/components/pagination/src/components/jumper.vue

```diff
@@ -37,14 +37,36 @@ const ns = useNamespace('pagination')
 const { pageCount, disabled, currentPage, changeEvent } = usePagination()
 const userInput = ref<number | string>()
 const innerValue = computed(() => userInput.value ?? currentPage?.value)
+let timer: ReturnType<typeof setTimeout> | undefined
 
 function handleInput(val: number | string) {
   userInput.value = val ? +val : ''
 }
 
 function handleChange(val: number | string) {
-  val = Math.trunc(+val)
-  changeEvent?.(val)
-  userInput.value = undefined
+  timer = setTimeout(() => {
+    val = Math.trunc(+val)
+    changeEvent?.(val)
+    userInput.value = undefined
+  }, 100)
```

Comment:
I think this change will make the code more difficult to maintain. Perhaps the event handle of input should be optimized.

Reply:
You're right.I will consider changing a way to fix this problem.